src/plugins/divideColor.js

```diff
@@ -3,9 +3,24 @@ import flattenColorPalette from '../util/flattenColorPalette'
 import withAlphaVariable from '../util/withAlphaVariable'
 
 export default function() {
-  return function({ addUtilities, e, theme, variants }) {
+  return function({ addUtilities, e, theme, variants, target }) {
     const colors = flattenColorPalette(theme('divideColor'))
 
+    if (target('divideColor') === 'ie11') {
+      const utilities = _.fromPairs(
+        _.map(_.omit(colors, 'default'), (value, modifier) => {
+          return [
+            `.${e(`divide-${modifier}`)} > :not(template) ~ :not(template)`,
+            { 'border-color': value },
+          ]
+        })
+      )
+
+      addUtilities(utilities, variants('textColor'))
```

Comment:
Hi @adamwathan 
Thanks for creating tailwind css.
I'm going through the source code to learn how it's built and I found that different values are being passed into `addUtilities` in this file.

```jsx
// line 19
addUtilities(utilities, variants('textColor'))

// line 37
addUtilities(utilities, variants('divideColor'))
```
Is there a reason that `variants('textColor')` should be used when `target('divideColor') === 'ie11'` is true? Thanks.

Reply:
Whoops that's a bug! I just pushed a release to fix it, thanks for pointing it out!